pywsitest/ws_response.py

```diff
@@ -74,9 +76,10 @@ def is_match(self, response: dict):
             (bool): True if the response matches based on the attributes
         """
         for key in self.attributes:
-            if key not in response:
+            resolved_value = get_resolved_value(response, key)
+            if resolved_value is None:
                 return False
-            # Check attribute values only if value isn't none
-            if self.attributes[key] is not None and response[key] != self.attributes[key]:
+
+            if self.attributes[key] is not None and self.attributes[key] != resolved_value:
```

Comment:
is "self.attributes[key] is not None" redundant? resolved_value can never be None.

Reply:
`self.attributes[key]` can be `None` if we're only asserting that a value exists, not what that value is.

Reply:
I meant resolved_value can never be None by the point it reaches this if condition, so the self.attributes[key] != resolved_value might be all you need to test at that point

Reply:
so 
`return resolved_value is not None and self.attributes[key] == resolved_value:
`